src/ucs/sys/topo/base/topo.c

```diff
@@ -357,15 +347,15 @@ static void ucs_topo_pci_root_distance(const char *path1, const char *path2,
     ucs_assertv(path_distance > 0, "path1=%s path2=%s", path1, path2);
 
     /* TODO set latency/bandwidth by CPU model */
-    distance->latency   = 300e-9;
-    distance->bandwidth = ucs_min(3500.0 * UCS_MBYTE,
-                                  (19200.0 * UCS_MBYTE) / path_distance);
+    distance->latency   = ucs_global_opts.pci_root_lat;
```

Comment:
maybe remove TODO?

Reply:
I think it is still relevant. However, we can customize it via `ucx.conf`.